src/Entities/ApiKey.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * This file is part of the PHP Rebilly API package.
+ *
+ * (c) 2015 Rebilly SRL
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Rebilly\Entities;
+
+use Rebilly\Rest\Entity;
+
+/**
+ * Class ApiKey
+ *
+ * ```json
+ * {
+ *   "id"
+ *   "description"
+ *   "datetimeFormat"
+ *   "userName"
+ *   "secretKey"
+ *   "createdTime"
+ * }
+ * ```
+ *
+ * @author Maksim Tuzov <maksim.tuzov@rebilly.com>
+ * @version 0.1
+ */
+final class ApiKey extends Entity
+{
+    /**
+     * @return string;
+     */
+    public function getDescription()
+    {
+        return $this->getAttribute('description');
+    }
+
+    /**
+     * @param string $value
+     *
+     * @return $this
+     */
+    public function setDescription($value)
+    {
+        return $this->setAttribute('description', $value);
+    }
+
+    /**
+     * @return string;
+     */
+    public function getDatetimeFormat()
+    {
+        return $this->getAttribute('datetimeFormat');
+    }
+
+    /**
+     * @param string $value
+     *
+     * @return $this
+     */
+    public function setDatetimeFormat($value)
+    {
+        return $this->setAttribute('datetimeFormat', $value);
```

Comment:
I think you can add constants for two possible datetime format and add some validation